File: php/TrafficLight.php
<?php
	//PHP is usually made for web so it is a uncommon to accept input from command line
	echo "Please input the light(red, yellow, and green)";
	$light = fgets(STDIN);
	//Comparison in PHP usually use == operator but for string comparison it is better to use strcmp method
	if(strcmp($light,"red")) {
		echo "Stop";
	} else if (strcmp($light,"yellow")) {
		echo "Stand By";
	} else if (strcmp($light,"green")) {
		echo "Run";
	} 
?>

File: go-lang/TrafficLight.go
package main

import "fmt"
import "bufio"
import "os"
import "strings"

func main() {
	reader := bufio.NewReader(os.Stdin);
	fmt.Println("Please input the light(red, yellow, and green)");

	light, _ := reader.ReadString('\n')
	//In this part is a bit tricky because at the end of the input contain "\n" 
	//so we need to trim it
	if strings.TrimRight(light, "\n") == "red" {
		fmt.Println("Stop");
	} else if strings.TrimRight(light, "\n") == "yellow" {
		fmt.Println("Stand By");
	} else if strings.TrimRight(light, "\n") == "green" {
		fmt.Println("Run");
	}
}